Controllers/ExportController.cs

```diff
@@ -158,38 +169,17 @@ public async Task<IActionResult> Export(int? id)
 
             var instances = await query.ListAsync();
 
-            var rows = instances.Select(x => GetOutput(x)).ToList();
-
-            var memoryStream = new MemoryStream();
-
-            var streamWriter = new StreamWriter(memoryStream);
-
-            var csvWriter = new CsvWriter(streamWriter);
-
-            var headers = rows.SelectMany(x => x.Keys).Distinct();
-
-            foreach (var header in headers)
+            try
             {
-                csvWriter.WriteField(header);
+                var stream = await _exportService.GetExportFileAsStreamAsync(instances);
```

Comment:
Think this is an unnecessary use of a variable, put it straight in to the first parameter on the next line.

Reply:
I think this passed my threshold for an acceptable length to sling straight into a parameter but if I make the `File` method call across multiple lines I think it'll be ok.